scim-sdk/src/main/java/com/unboundid/scim/wink/BulkResource.java

```diff
@@ -1,4 +1,4 @@
-/*
+ /*
```

Comment:
Tiny little thing, but it looks like you accidentally inserted a space right here.